cranelift/codegen/src/isa/aarch64/lower.rs

```diff
@@ -2753,7 +937,7 @@ impl LowerBackend for AArch64Backend {
     type MInst = Inst;
 
     fn lower<C: LowerCtx<I = Inst>>(&self, ctx: &mut C, ir_inst: IRInst) {
-        lower_insn_to_regs(ctx, ir_inst);
+        lower_inst::lower_insn_to_regs(ctx, ir_inst);
     }
 
     fn lower_branch_group<C: LowerCtx<I = Inst>>(
```

Comment:
If the file-size limits allow it, it might be good to move the branch-lowering to `lower_inst.rs` as well, just to keep all of the opcode-switches together (or if not, perhaps a separate `lower_branch.rs`).

Reply:
Yep, up to 80K chars, but still under the 100K limit, done.